site/custom.css

```diff
@@ -0,0 +1,3 @@
+:root {
+    --sl-content-width: 100rem;
```

Comment:
perhaps the most controversial change!

Reply:
Yeah, this ends up nuking the helpful navigation for the markdown docs:
Existing
![image](https://github.com/user-attachments/assets/ee012893-a0ac-49ef-83cb-6b551d862e5c)

New
![image](https://github.com/user-attachments/assets/f112cf8c-9e6c-4aa9-8261-a7dd84a3bbe6)

I love the idea of making the examples bigger, let's just scope this change to only those ones. I think we can accomplish that with a custom layout for those pages